### DISCUSSION TITLE
Support having multiple workspaces open in different windows, and for a workspace to contain multiple windows

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#576] [Feature Request]
I was going to write this FR myself but someone on Reddit [already did](https://www.reddit.com/r/Floorp/comments/16u2v43/workspaces_only_showing_in_one_window/):

> Hi, I'm trying to figure out a good workflow to use Floorp for managing multiple projects/areas of interest. I was impressed to see that workspace functionality exists, but when I open a new window, I lose access to the workspaces button. Additionally if I switch workspaces while the additional window is open, nothing changes. The workspace functionality only works if you use your browser with only one window. I tend to have a few projects going at once and switch between them periodically throughout the day. I would love to be able to have workspaces dedicated to projects and areas of interest, and have multiple workspaces open simultaneously. The way vivaldi does it make sense - you have an option to open a workspace in a new window if you want to, and when you clsoe the workspace, you can open it back up again anytime you want.

**Comment by neurodiverseEsoteric** [comment]
https://github.com/Floorp-Projects/Floorp/issues/768 is a design choice that makes me regret switching to floorp. It would be extremely helpful—and a good accessibility feature—To be able to view and work on multiple workspaces through separate windows, and/or running multiple workspaces across (linux) desktops, at the same time. Especially when one has hundreds of tabs per hundreds of workspaces...
